### DISCUSSION TITLE
Can't Access Atlas

### DISCUSSION THREAD

**Discussion by davemowery** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#30] [General]
I can't seem to get to the Atlas. Polo will give coordinates, but I can't see Atlas on my galaxy map.

**Comment by cwmonkey** [comment]
@davemowery Ask them for black hole coordinates and they will give you both black hole AND Atlas station coordinates. This should be mentioned in a few readme's, but if it's not in any of the places you looked, re-open this and we will put it on the to-do list.
